client.go

```diff
@@ -178,6 +178,9 @@ type RemoteContext struct {
 	// Snapshotter used for unpacking
 	Snapshotter string
 
+	// Labels to be applied to pulled references
```

Comment:
Clarify that this only applies to the image record and not transitive resources.